experimental/table/client/pom.xml

```diff
@@ -35,6 +35,10 @@
       <groupId>io.netty</groupId>
       <artifactId>netty-all</artifactId>
```

Comment:
so do we still need this `netty-all` dependency?

Reply:
Yes, `netty-all` includes `netty-native`, but we need to explicitly add `netty-native` because we exclude `netty-all` in Presto. The way they define netty dependencies is each module individually (whereas we just include all), this means for the packages in `all` that are not included in Presto, we need to explicitly depend on them.

Reply:
got it